tasks/instance_users.yml

```diff
@@ -32,5 +33,6 @@
     password: "{{ item.fenced_password | default('ldUMKEu7/QCAY')}}"
     home: "{{ item.fenced_home_directory | default(omit) }}"
     uid: "{{ item.fenced_uid | default(omit) }}"
+    createhome: "{{ item.createhome | default(omit) }}"
```

Comment:
Since this is a boolean and the default for this variable is `true` I think it would be more readable if we declare `default(true)` instead of omitting (which will make ansible create the directory). Does that make sense to you?

Reply:
makes sense. Even if the default is true, it will be more resilient to future changes if they occur. I'll change this one as you suggest.